python/src/py/keyvi/dictionary/__init__.py

```diff
@@ -17,4 +17,4 @@
 limitations under the License.
 '''
 
-from keyvi._core import Dictionary
+from keyvi._core import Dictionary, loading_strategy_types
```

Comment:
@hendrikmuhs I expect later on `keyvi.vector` will support `loading_strategy_types` as well, maybe it's worth to expose it on `global` level only ?

Reply:
ok, makes sense